mdpow/tests/test_workflows_base.py

```diff
@@ -21,6 +23,19 @@ def molname_workflows_directory(tmp_path):
 
 class TestWorkflowsBase(object):
 
+    check_guessed_elements = np.array(['C', 'N', 'C', 'Cl', 'C'])
+
+    @pytest.fixture(scope='function')
+    def create_universe(self):
```

Comment:
I call fixtures that provide a thing "thing" — the creation part is technical stuff.
```suggestion
    def universe(self):
```